src/strategies/flashstake/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getAddress } from '@ethersproject/address';
+import { formatUnits } from '@ethersproject/units';
+import { multicall, subgraphRequest } from '../../utils';
+
+const FLASHSTAKE_SUBGRAPH_URL = {
+  '1':
+    'https://api.thegraph.com/subgraphs/name/blockzerohello/flash-stake-stats-v2-subgraph'
+};
+
+export const author = 'BlockzeroLabs';
```

Comment:
This must be your GitHub username
```suggestion
export const author = 'anassohail99';
```